.github/workflows/migrate.yml

```diff
@@ -98,5 +102,5 @@ jobs:
             fi
           done
           
-          juju switch default
+          juju switch test-migrate
           juju wait-for application ubuntu
```

Comment:
Oh and one more thing. To further verify the migration is successful we should consider deploying an application

We have had it before that a model has appeared to migrate successfully, but has been borked and non-functional